pylint/lint/pylinter.py

```diff
@@ -600,7 +600,7 @@ def should_analyze_file(modname: str, path: str, is_argument: bool = False) -> b
         """
         if is_argument:
             return True
-        return path.endswith(".py")
+        return path.endswith(".py") or path.endswith(".pyi")
```

Comment:
```suggestion
        return path.endswith(("py", ".pyi"))
```
Either way is fine I just thought I'd throw the idea out there :)